src/groups/mqb/mqbblp/mqbblp_cluster.cpp

```diff
@@ -3605,8 +3628,6 @@ Cluster::sendRequest(const Cluster::RequestManagerType::RequestSp& request,
                      mqbnet::ClusterNode*                          target,
                      bsls::TimeInterval                            timeout)
 {
-    // executed by the cluster *DISPATCHER* thread
```

Comment:
Yes, it is now possible to be invoked in the queue dispatcher thread by `mqbc::StorageManager`: https://github.com/bloomberg/blazingmq/blob/751032b99ad8dbf95c67dbdd696fb6919ef8dd74/src/groups/mqb/mqbc/mqbc_storagemanager.cpp#L1742

Reply:
Added some comment about thread safety. Importantly, `Cluster::sendRequest()` has been invoked in queue dispatcher threads since before FSM, for example at [`RecoveryManager::onPartitionSyncStateQueryResponseDispatched()`](https://github.com/bloomberg/blazingmq/blob/8d133d6510cddc936e2ab2492cba189f009c2add/src/groups/mqb/mqbblp/mqbblp_recoverymanager.cpp#L2617-L2620).